packages/site/src/ui/link.js

```diff
@@ -27,7 +29,10 @@ const Link = React.forwardRef((props, ref) => {
     </NextLink>
   )
 })
+
 Link.appearances = DSLink.appearances
+Link.displayName = 'Link'
```

Comment:
Nice catch.  How in the world did you discover that amongst everything else?

Reply:
there are linting warnings around components missing display names. i'd like to eventually enforce that rule once everything is in compliance. while pairing with teams i've noticed that it's super hard to use the react dev tools because a ton of components are just named `Unknown`